src/docfx/lib/ProcessUtility.cs

```diff
@@ -45,32 +45,36 @@ public static async Task<bool> IsExclusiveLockHeld(string lockName)
         /// Acquire a shared lock for input lock name
         /// The returned `acquirer` are used for tracking the acquired lock, instead of thread info, since the thread info may change in asynchronous programming model
         /// </summary>
-        public static async Task<(bool acquired, string acquirer)> AcquireSharedLock(string lockName)
+        public static Task<(bool acquired, string acquirer)> AcquireSharedLock(string lockName, bool retryUntilSuccess = false)
```

Comment:
any test coverage for the input `retryUntilSuccess` true?

Reply:
yes, changed